Add trigger selector for blueprint (#29598)
home-assistant/home-assistant.github.io: source/_docs/blueprint/selectors.markdown

The output of this selector will contain the time in 24-hour format,
for example, `23:59:59`.

## Trigger selector

The triggers selector allows the user to input one or more triggers.

> On the user interface, the trigger part of the automation editor will be shown.

Reviewer: ```suggestion
On the user interface, the trigger part of the automation editor is shown.
```

> The value of the input will contain a list of triggers.

Reviewer: ```suggestion
The value of the input contains a list of triggers.
```
